llvm/test/Transforms/LoopVectorize/pr66616.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: opt -passes="require<scalar-evolution>,print<scalar-evolution>,loop-vectorize" --verify-scev --verify-scev-strict -S < %s | FileCheck %s
+
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
Does this test require the x86 target? If so, it needs to be moved to the `X86` sub-directory. But it is probably sufficient to pass something like `-force-vector-width=4 -force-vector-interleave=1`  to force vectorization without target triple.